src/definitions/collections/form.less

```diff
@@ -1117,4 +1117,13 @@
   font-size: @massive;
 }
 
+
+.ui.inverted.form textarea.transparent,
+.ui.form .inverted.segment textarea.transparent {
+  color: inherit;
+}
+
```

Comment:
Why so much whitespace?

Reply:
I simply adjusted the logic the same as it is for input.

... or maybe i don't get, what you mean? :thinking:

Reply:
if you simply mean the additional empty lines...then i am simply sorry for that, there was no reason for sure

Reply:
What he probably meant is that you added two empty lines before the selector and four lines after.
Additionally couldn't this be added to this:
https://github.com/fomantic/Fomantic-UI/blob/31ed0007b7133967b7ef2ad2d52cc8328acf4299/src/definitions/collections/form.less#L162-L173
or to the inverted variation section?
https://github.com/fomantic/Fomantic-UI/blob/31ed0007b7133967b7ef2ad2d52cc8328acf4299/src/definitions/collections/form.less#L714-L749

Reply:
You have 4 blank lines of whitespace added and there isn't any need for it

Reply:
ok, got it now working in the suggested transparent section also. see latest commit. Sorry for any inconvenience :wink: